selvpcclient/resell/v2/crossregionsubnets/schemas.go

```diff
@@ -0,0 +1,54 @@
+package crossregionsubnets
+
+import (
+	"github.com/selectel/go-selvpcclient/selvpcclient/resell/v2/servers"
+)
+
+// CrossRegionSubnet represents a single Resell cross-region subnet.
+type CrossRegionSubnet struct {
+	// ID is a unique id of a cross-region subnet.
+	ID int `json:"id"`
+
+	// CIDR is a cross-region subnet prefix in CIDR notation.
+	CIDR string `json:"cidr"`
+
+	// VlanID represents id of the associated VLAN in the Networking service.
+	VlanID int `json:"vlan_id"`
+
+	// Status shows if cross-region subnet is used.
+	Status string `json:"status"`
+
+	// Servers contains info about servers to which cross-region subnet is associated to.
+	Servers []servers.Server `json:"servers"`
+
+	// Subnets contains info about subnets in every region that cross-region subnet is attached to.
+	Subnets []Subnet `json:"subnets"`
+}
+
+// Subnet represents a single subnet to which cross-region subnet is associated to.
+type Subnet struct {
```

Comment:
It's better to remove it and add `VLANID` and `VTEPIPAddress` fields into the existing `subnets.Subnet structure`.
It doesn't really matter if their values will be populated only from `cross_region_subnets` API controller responses.